src/routes/controllers/articles.controller.js

```diff
@@ -331,3 +345,11 @@ export async function removeBookmark(req, res) {
     return errorResponse(res, 500, error.message);
   }
 }
+
+export const recordARead = async (articleId, user = null) => {
+  let userId;
+
+  user ? (userId = user.id) : (userId = null);
```

Comment:
I would prefer you use normal if or else statements here instead of ternary operator for readability.

Reply:
I have fixed it